cfme/tests/v2v/test_v2v_ansible.py

```diff
@@ -21,7 +20,7 @@
         server_roles=["+embedded_ansible"]
     ),
     pytest.mark.provider(
-        classes=[RHEVMProvider],
+        classes=[OpenStackProvider, RHEVMProvider],
```

Comment:
@sshveta  As I understand, this test now should run on OpenStack as well, can you add it to PRT?

Reply:
Hi Angeline , OSP has some provider issue in rest api which is being investigated by @Yadnyawalkya . Till then we can execute this only for RHV